core/src/main/scala/scalaz/ISet.scala

```diff
@@ -628,7 +628,7 @@ sealed abstract class ISet[A] {
 sealed abstract class ISetInstances {
   import ISet._
 
-  implicit def setEqual[A: Equal]: Equal[ISet[A]] = new ISetEqual[A] {
+    implicit def setEqual[A: Equal]: Equal[ISet[A]] = new ISetEqual[A] {
```

Comment:
could you please keep this PR only about the fix, and split any refactoring or cosmetic changes?